Art Institute of Chicago.js

```diff
@@ -0,0 +1,222 @@
+{
+	"translatorID": "8e98b11a-5648-42b2-8542-5f366cb953f6",
+	"label": "Art Institute of Chicago",
+	"creator": "nikhil trivedi, Illya Moskvin",
+	"target": "^https?://(www.)?artic.edu/artworks/.*",
```

Comment:
This doesn't need the .* at the end but you should escape the other `.`

Reply:
Got it.